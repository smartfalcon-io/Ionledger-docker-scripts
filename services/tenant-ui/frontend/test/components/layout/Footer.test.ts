import { mount } from '@vue/test-utils';
import { describe, expect, test } from 'vitest';

import Footer from '@/components/layout/Footer.vue';

describe('Header', async () => {
  test('mount matches snapshot with expected values', () => {
    const wrapper = mount(Footer);

    expect(wrapper.html()).toMatchSnapshot();
  });
});
