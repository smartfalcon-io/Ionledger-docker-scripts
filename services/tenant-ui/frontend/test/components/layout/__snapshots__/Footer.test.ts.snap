// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Header > mount matches snapshot with expected values 1`] = `
"<div class="traction-footer flex justify-content-between flex-wrap">
  <div>dashboard.greeting</div>
  <div> </div>
</div>"
`;
